packages/breadcrumbs/stories/breadcrumbs.stories.ts

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import { html, TemplateResult } from '@spectrum-web-components/base';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-settings.js';
+
+import {
+    getBreadcrumbs,
+    getBreadcrumbsWithLinks,
+    getResizableStyles,
+    StoryArgs,
+    Template,
+} from './template.js';
+import { argTypes } from './args.js';
+import { ifDefined } from '@spectrum-web-components/base/src/directives.js';
+import { spreadProps } from '../../../test/lit-helpers.js';
+
+export default {
+    title: 'Breadcrumbs',
+    component: 'sp-breadcrumbs',
+    args: {
+        'max-visible-items': 4,
+        nrOfItems: 4,
```

Comment:
Toggling the `nrOfItems` does not produce effect for me in any story.

Reply:
(nor in the documentation website preview)

Reply:
Yeah... I added support for it just in the "Collapsible behaviour" story. I can add in all of them, but then the "Collapsible behaviour" story will not bring that much of a value :(

For Docs, this actually is a limitation here, I can only add plain HTML in the docs preview, it will not render at all if I add a function here which does some `map` iterations and conditionally renders the nr you enter in `nrOfItems`.
I actually struggled a couple of hours of why my docs are not loading, and it was the use of [`getBreadcrumbs` ](https://github.com/adobe/spectrum-web-components/pull/4578/files#diff-0c0142ceb4a56897b37fa7182a4a4c0647bef74e019e39e5f2061096c05ecd73R53) method that was breaking it.

Reply:
I ended up removing this control. It was useful for testing child addition and removal while in development and testing, but I guess it does not bring much value to consumers and it might be confused which an actual property you could apply.
Instead, you can play with the `max-visible-items` control to get a similar behaviour of breadcrumbs expanding / collapsing.
Regarding the docs, I opened a different PR to fix the issue with changes not being applied there #4636.